### DISCUSSION TITLE
Machine Specs

### DISCUSSION THREAD

**Discussion by hra42** [community/community#52408] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I wanted to select a higher machine type (for example the 8 core machine) but I won't even shown the option neither with a new nor allready created codespace. 

I am on the Github Pro Plan and have billing and spending limit setup.

I also applyed for the gpu machine, but havn't heard for months on that...

**Comment by cmuto09** [comment]
Hi @DNS-HRA- thanks for reaching out! I can enable you for more powerful CPU-based machines (up to 32 cores). Which would you need access to? 

For GPU-based machines, we have a limited capacity of machines available in the beta, so activations have been taking longer than anticipated. Let me get back to you regarding access to those machines, as I will need to review our current capacity ahead of enabling that.

Thanks!

**Reply by hra42** [reply]
@cmuto09 any Update on that?

**Comment by hra42** [comment]
Hi @cmuto09 I would at least need the 8 and 16 core machines. The 32 cores would be a great tool for larger builds, but if there is no capacity, that's no problem.

I just asked for a status of the GPU based machines because I signed up for them when they were announced but never heard back…

**Reply by cmuto09** [reply, marked as answer]
Hey @DNS-HRA- I have enabled you for 8, 16, and 32-core machines. You should be able to begin using those new machine types immediately. Regarding GPUs, unfortunately I am not yet able to enable that due to capacity. I will update as anything changes.

In the meantime, enjoy the additional horsepower, and please reach out with any feedback as you test the larger Codespaces!

**Reply by dostuffthatmatters** [reply]
Hi @cmuto09,

I would like to use a GPU as well, but the survey under "request bigger machines" does not exist anymore. Could you please add me to the waiting list as well, if that's possible?

Best,
Moritz

**Comment by foysal619** [comment]
If you are not seeing the option to select a higher machine type in GitHub Codespaces, there are a few things you can try:

Make sure you have selected a repository or created a new one: GitHub Codespaces are tied to specific repositories, so you will need to select a repository or create a new one before you can see the option to select a machine type.

Check your account settings: Make sure you have a valid payment method on file and that your billing and spending limits are set up correctly. You can check this by going to your account settings and clicking on the "Billing & plans" tab.

Try clearing your cache: If you have recently made changes to your account settings, you may need to clear your cache in order to see the changes reflected in your Codespaces.

If none of these solutions work, you can contact GitHub support for further assistance. As for the GPU machine, GitHub Codespaces currently only support CPU machines, so it is possible that your application for a GPU machine has not been processed yet. You can contact GitHub support to check the status of your application.

**Reply by hra42** [reply]
> If you are not seeing the option to select a higher machine type in GitHub Codespaces, there are a few things you can try:
> 
> Make sure you have selected a repository or created a new one: GitHub Codespaces are tied to specific repositories, so you will need to select a repository or create a new one before you can see the option to select a machine type.
> 
> Check your account settings: Make sure you have a valid payment method on file and that your billing and spending limits are set up correctly. You can check this by going to your account settings and clicking on the "Billing & plans" tab.
> 
> Try clearing your cache: If you have recently made changes to your account settings, you may need to clear your cache in order to see the changes reflected in your Codespaces.
> 
> If none of these solutions work, you can contact GitHub support for further assistance. As for the GPU machine, GitHub Codespaces currently only support CPU machines, so it is possible that your application for a GPU machine has not been processed yet. You can contact GitHub support to check the status of your application.

I have done all that allready. First with Github Pro I have 180 houres per month for free anyway. A credit card that can be balanced is allready assigned to the account. I created a completly new repo. I checked via the API and saw that the machines type arn't available. If I read the answer from @cmuto09 right it needs to be enabled. I contacted Github Support too, but got no answer until now.

**Comment by Someet-Git** [comment]
Hi @cmuto09 
I would like to upgrade my codespace to at least 16 core machines and want to use the GPU(if possible), but the survey website doesn't exist anymore. Can you help me with this.

Thanks

**Comment by MounZ17** [comment]
hey @cmuto09 
i would like to upgrade my codespace to 16 or 32 core machines if possible, i'm currently using it to train a model with a large amount of data. can you please help me with it ?
thank you !

**Comment by Nidvogr** [comment]
Hey, I've got a 'pro account' due to the 'student pack' from github, but I can't create any GPU machines to do machine learning on? I've tried contacting support, but they sent me here instead. I guess all the accounts need to be manually enabled for GPU? If so, is it possible for you to enable it?
Kind regards

**Reply by fretchen** [reply]
The same applies to me. Would it be possible to obtain access to a gpu machine for codespaces ?

**Comment by jtx1999** [comment]
Hi @cmuto09 
I would like to upgrade my codespace to 8, 16 and 32 core machines as well. Currently I can only run with 2 or 4 cores. Could you please help me with that?
Thanks!

**Comment by kleinlennart** [comment]
Still the same issue. Any updates on this?

**Reply by hra42** [reply]
You should write to GitHub Support about this ;)
